packages/react/src/themes/teams/components/Dropdown/dropdownStyles.ts

```diff
@@ -106,7 +106,7 @@ const dropdownStyles: ComponentSlotStylesPrepared<DropdownPropsAndState, Dropdow
     overflowY: 'auto',
     maxHeight: v.selectedItemsMaxHeight,
     width: '100%',
-    ...(p.toggleIndicator && { paddingRight: v.toggleIndicatorSize }),
+    ...(p.toggleIndicator && { marginRight: v.toggleIndicatorSize }),
```

Comment:
this actually breaks IE in a strange way. Follwoing code added to the toggle indicator slot works better:
```
 root: {
   [`& .${Dropdown.slotClassNames.toggleIndicator}`]: {
     marginRight: "12px"
   }
  }
```

Reply:
@jurokapsiar what is the strange behavior?

Reply:
It looks like the new styling moves the scrollbar for all browsers and not just IE11. I don't think that is what we will want, but i can't load in ie11 right nowso i don't understand the issue...